exercises/practice/leap/.approaches/conditional-expression/content.md

```diff
@@ -0,0 +1,118 @@
+# Conditional expression
+
+```haskell
+isLeapYear :: Integer -> Bool
+isLeapYear year =
+  if divisibleBy 100
+    then divisibleBy 400
+    else divisibleBy 4
+  where
+    divisibleBy d = year `mod` d == 0
+```
+
+
+## Conditional expressions
+
+A _conditional expression_ (`if … then … else …`) is a compound expression that uses a test to determine which of two alternatives to evaluate to.
+Many other languages feature a similar construct, often termed 'ternary operator'.
+They are also known as _`if` expressions_.
+
+Concretely, when `p` is some expression of type `Bool` and `t` and `f` are any two expressions of the same type, then `if p then t else f` will
```

Comment:
I notice there were a few sentences that began with _Concretely, ..._. Personally, I think dropping this makes the text easier to read. For example:

```suggestion
When `p` is some expression of type `Bool` and `t` and `f` are any two expressions of the same type, then `if p then t else f` will
```